### DISCUSSION TITLE
Importar clientes

### DISCUSSION THREAD

**Discussion by DataPowerTI** [RamonSilva20/mapos#2120] [General]
Bom dia pessoal, existe alguma maneira de importar clientes do sistema que eu uso atualmente para o MAPOS?
Tipo eu exportar do sistema antigo para excel e importar no MAPOS.

**Comment by lukasabino** [comment]
Boa noite,

tem como importar com um arquivo CSV, mas direto no banco de dados... Pelo painel do Mapos em si, não tem como.

**Reply by DataPowerTI** [reply]
sabe mo dizer como eu faço isso?
tenho qu criar a tbela a ser importada em algum modelo específico? com campos em ordem específicas?

**Comment by willph** [comment]
> Bom dia pessoal, existe alguma maneira de importar clientes do sistema que eu uso atualmente para o MAPOS? Tipo eu exportar do sistema antigo para excel e importar no MAPOS.

Esse sistema antigo é MapOS também? ou é outro sistema?

**Reply by DataPowerTI** [reply]
Era outro sistema, acabei adicionando manualmente mesmo.
